plugin.xml

```diff
@@ -71,6 +71,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
     <!-- android -->
     <platform name="android">
         <config-file target="res/xml/config.xml" parent="/*">
+            <preference name="AndroidXEnabled" value="true" />
```

Comment:
This preference is being removed in the next release of Cordova Android.

As this PR is implementing with Android X, it will have to be released in next major of this plugin and flagged for next major of Android.

This can be removed...

Reply:
But I think the app that is used for the automated tests is using the old version. Removing this flag will fail the tests.

Reply:
> But I think the app that is used for the automated tests is using the old version. Removing this flag will fail the tests.

I think we can leave this in for now just for the sake of using the automated tests. Once `cordova-android@10` is released, it can be removed, as the automated tests will start using `cordova-android@10`, which should work without adding this preference.

We'll need to also update the engines on this plugin so that older versions of plugin cannot be used on `cordova-android@`10`, and version 4 of this plugin is required if using `cordova-android@10`, but this can be done as a separate independent PR.

Reply:
Agreed.

Reply:
I am OK with it staying in for the passing of the tests but this PR wont be merged in until then.

Reply:
Would it be merged if the travis automated tests fail? AFAIK in August it is required for Android app to target Android 11, and by then this plugin will not function properly.

Reply:
This type of PR can not be merged in until after Cordova Android 10 is released because of the hard requirement to use AndroidX library.

If you want this merged into the main branch and able to use it with the current Cordova Android 9.x release, you will need to remove all of the AndroidX requirements and go back to the ASL.

Reply:
I think using the old Android library would be a good enough solution for now, we can have another PR to update it to AndroidX when cordova-android 10 is released.